src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -75,10 +80,39 @@ public CommandResult addPersonTags(Model model) throws CommandException {
         personToEdit.setCommonModules(userModuleTags);
 
         model.updateObservablePersonList();
+
+        //
+        // need to take the set of module tags in this command
+        // loop through and then put into the
+        for (ModuleTag tag : moduleTags) {
```

Comment:
Not sure if anyone can direct me to a cleaner way of doing this other than a for loop.

Reply:
Not sure if you want to separate out the logic in the for loop into another method and use streams to apply it using the `forEach` method.

Reply:
It’s not that we cannot use for loops, it’s just that if you’re going to heavily nest it, then it becomes a problem.